taxonomies/ingredients_processing.txt

```diff
@@ -932,7 +932,7 @@ uk:сушена
 en:semi-dried
 de:halbgetrocknet, halbgetrocknete, halbgetrockneter, halbgetrocknetes, halb getrocknet
 es:semiseco
-fr:semi séché, semi-séchée, semi-séchés, semi-séchées
+fr:semi séché, semi-séchée, semi-séchés, semi-séchées, miséchées
```

Comment:
miséchées without a - (mi-séchées) is a spelling / OCR mistake I think. We do have a few products with "mi-séché" and variations.

```suggestion
fr:semi séché, semi-séchée, semi-séchés, semi-séchées, mi-séché, mi-séchée, mi-séchés, mi-séchées
```